WORKSPACE

```diff
@@ -133,7 +133,7 @@ container_pull(
     digest = "sha256:e357a240b1e8668c851a1dcb3e3244b4e037aab56029f394dda6925070d473a9",
```

Comment:
This SHA must be updated too.

Reply:
https://console.cloud.google.com/gcr/images/pipecd/GLOBAL/piped-base@sha256:e863054859305080e89741bad1444e46a05ecb57cf2faf33a4bd448a992c47f6/details?tab=info

Reply:
Thanks :D